examples/nsxt/avi_nsxt_cloud/nsx-t-cloud.tf

```diff
@@ -0,0 +1,95 @@
+provider "avi" {
+  avi_username   = var.avi_username
+  avi_tenant     = var.avi_tenant
+  avi_password   = var.avi_password
+  avi_controller = var.avi_controller
+  avi_version    = var.avi_version
+}
+
+provider "vsphere" {
+  user           = var.vsphere_user
+  password       = var.vsphere_password
+  vsphere_server = var.vsphere_server
+
+  # If you have a self-signed cert
+  allow_unverified_ssl = true
+}
+
+provider "nsxt" {
+  host                     = var.nsxt_url
+  username                 = var.nsxt_username
+  password                 = var.nsxt_password
+  allow_unverified_ssl     = true
+}
+
+#Retrieve NSXT Transport Zone UUID
+#Remove nsx-tr-zone if you have multiple transport zones with same name
+data "nsxt_transport_zone" "nsx-tr-zone" {
+  display_name   = var.transport_zone_name
+}
+
+#Retrieve vSphere Content Library UUID
+data "vsphere_content_library" "library" {
+  name           = var.content_library_name
+}
+
+#Creates NSX-T User on Avi controller
+resource "avi_cloudconnectoruser" "nsx-t-user" {
+  name           = "nsx-mgr-tf-user"
```

Comment:
shall we not read this as well from the variable?

Reply:
I will change this to variable.